scalafix-tests/input/src/main/scala/test/redundantTokens/FinalObject.scala

```diff
@@ -0,0 +1,11 @@
+/*
+rules = RedundantTokens
+RedundantTokens.finalObject = true
```

Comment:
some negative tests here where the final keyword is used on other symbols would be nice against regressions